### ISSUE TITLE
Accessibility issues with MediaPlayerElement.MediaTransportControls 

### ISSUE THREAD

**Issue by mrlacey** [microsoft/microsoft-ui-xaml#2994]
**Describe the bug**

The MediaTransportControl contains a default value that causes it to fail basic accessibility testing. Specifically, there is an unnamed progress bar that is exposed to the UI Automation Tree without a name.
The only workaround for this is to completely implement custom transport controls.

**Steps to reproduce the bug**

1. Add MediaPlayer control to a page
2. Include a `MediaTransportControls` instance as the MediaTransport property for the MediaPlayerElement.

```xml
    <MediaPlayerElement AreTransportControlsEnabled="True">
        <MediaPlayerElement.TransportControls>
            <MediaTransportControls IsCompact="False" />
        </MediaPlayerElement.TransportControls>
    </MediaPlayerElement>
```

3. Run app
4. Test the app with Accessibility Insights for Windows

**Expected behavior**

The default behavior of the control should not contain anything that causes it to fail basic accessibility checks.
Specifically [Section 508 502.3.1](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology).

**Screenshots**

![image](https://user-images.githubusercontent.com/189547/88541946-7b4cbb80-d00d-11ea-9e26-4750bb47b4a3.png)

**Version Info**

NuGet package version: Microsoft.NETCore.UniversalWindowsPlatform 6.2.10

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (20175)              | Yes   |
| May 2020 Update (19041)            |  Yes   |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            |  Yes |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Additional context**

This was discovered while reviewing the accessibility of applications generated by Windows Template Studio. (https://github.com/microsoft/WindowsTemplateStudio/issues/3723)

**Comment by StephenLPeters** [comment]
@Austin-Lamb Do you know which team this should be routed to?

**Comment by karenbtlai** [comment]
Media Player Element now passes all accessibility insight tests with updated styling, so closing.
